packages/@sanity/desk-tool/src/panes/document/documentHistory/history/Controller.ts

```diff
@@ -267,8 +267,10 @@ export class Controller {
     }
 
     const url = `/data/history/${dataset}/transactions/${publishedId},${draftId}?${queryParams}`
+    const projectId = this.client.config().projectId
+    const token = projectId ? getToken(projectId) : null
 
-    const stream = await getJsonStream(this.client.getUrl(url))
+    const stream = await getJsonStream(this.client.getUrl(url), token || undefined)
```

Comment:
would it make sense to grab the token from the client? will the client ever have a different token then the global token?

Reply:
It will not be different one no. This is a nice simplification, I'll get it from the client instead.